src/get-started/java-binding.md

```diff
@@ -240,6 +240,42 @@ at `/cryptocurrency/balance/:walletId`.
 See [documentation][vertx.io] on the possibilities of `Vert.x` used as a web
 framework.
 
+### Built-In Services
```

Comment:
I'd agree with Oleg that this section location is out of place. I think it must be placed either after "How to Run a Service" on the same level or _in_ "How to Run a Service" on the level below. I'd prefer the former.